mlflow/utils/databricks_utils.py

```diff
@@ -187,6 +187,14 @@ def is_in_databricks_model_serving_environment():
     return val.lower() == "true"
 
 
+def enable_mlflow_tracing() -> bool:
+    """
+    This environment variable guards tracing behaviors for models in databricks
+    model serving. Tracing in serving is only enabled when this env var is true.
+    """
+    return os.environ.get("ENABLE_MLFLOW_TRACING", "false").lower() == "true"
```

Comment:
can we move this in `mlflow/environment_variables.py`?

Reply:
or is there a reason we can't do this?

Reply:
I'd prefer move this if in the future we want customers to set it, for now I think it's internally set.

Reply:
Just to confirm, If I want to enable tracing in model serving, I need to set `ENABLE_MLFLOW_TRACING=true` to my endpoint, correct?

Reply:
Yes